### PR TITLE
Render ignored users setting regardless of if there are any

### PR DESCRIPTION
**Pull request by samhh**
Closes https://github.com/vector-im/element-web/issues/14968.

### CODE REVIEW

**Comment by SimonBrandner** [comment]
@samhh, thanks for working on this! Could you please include a sign-off as described [here](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.rst#sign-off)?

**Comment by samhh** [comment]
I knew I'd forgotten something! :slightly_smiling_face: I've rebased with commit sign-off. First time I've used that Git feature!

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.js [hunk 1]
```diff
@@ -255,10 +255,9 @@ export default class SecurityUserSettingsTab extends React.Component {
     _renderIgnoredUsers() {
         const {waitingUnignored, ignoredUserIds} = this.state;
 
-        if (!ignoredUserIds || ignoredUserIds.length === 0) return null;
-
-        const userIds = ignoredUserIds
-            .map((u) => <IgnoredUser
+        const userIds = !ignoredUserIds?.length
+            ? 'You have no ignored users.'
```
Comment (turt2live) [inline src/components/views/settings/tabs/user/SecurityUserSettingsTab.js:259]: This needs to be a translated string (`_t` and `yarn i18n`)

**Review by turt2live — CHANGES_REQUESTED** [review]
can we also get a screenshot please to show what it looks like in context? Thanks!

**Comment by samhh** [comment]
2021-04-13-214216_826x379_scrot

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.js (hunk 1 above)
Comment (samhh) [reply src/components/views/settings/tabs/user/SecurityUserSettingsTab.js]: Made the change. I've amended the commit as it's still a very small diff.

**Review by turt2live — APPROVED** [review]

**Pull request merged**